lib/views/accordion.js

```diff
@@ -0,0 +1,85 @@
+import React, {Fragment} from 'react';
+import PropTypes from 'prop-types';
+
+import {autobind} from '../helpers';
+
+export default class Accordion extends React.Component {
+  static propTypes = {
+    leftTitle: PropTypes.string.isRequired,
+    rightTitle: PropTypes.string,
+    results: PropTypes.arrayOf(PropTypes.any).isRequired,
+    isLoading: PropTypes.bool.isRequired,
+    loadingComponent: PropTypes.func,
+    emptyComponent: PropTypes.func,
+    children: PropTypes.func.isRequired,
+  };
+
+  static defaultProps = {
+    loadingComponent: () => null,
+    emptyComponent: () => null,
+  };
+
+  constructor(props) {
+    super(props);
+    autobind(this, 'toggle');
+
+    this.state = {
+      expanded: true,
+    };
+  }
+
+  render() {
+    return (
+      <details className="github-Accordion" open={this.state.expanded}>
```

Comment:
This seems to not work for the first click?

1. Reload Atom -> `open` is present
2. Click on the header -> `open` stays present
3. Click again on the header -> `open` gets removed

![open](https://user-images.githubusercontent.com/378023/41400689-c2dd3afa-6ff8-11e8-980a-1e03b71165eb.gif)

Reply:
Ahhh good eye. Looks like I was missing a `.preventDefault()` call.

Reply:
Oh, sure. I do like keeping markup semantic when I get the chance and not just using `<div>` every time. But if it's more convenient we can make the elements whatever's easiest.